modules/ROOT/pages/bpm-api.adoc

```diff
@@ -3,5150 +3,9 @@
-A JSON representation of a list of timer event triggers, as described above
- ** *Code*: 200
- ** *Payload*:
-+
-[source,json]
-----
-[
-  {
-    "id":4015,
-    "id_string":"4015",
-    "eventInstanceId":2,
-    "eventInstanceId_string":"2",
-    "executionDate":1413980484194,
-    "eventInstanceName":"Minuterie1"
-  }
-]
-----
-
-==== Update a timer event trigger next execution date
-
-Specify the next execution date of a timer event trigger.
-
-* *URL* +
-`/API/bpm/timerEventTrigger/:timerEventTriggerID`
-* *Method* +
-`PUT`
-* *Request Payload* +
-A JSON representation of a long value with attribute name "executionDate"
-+
-[source,json]
-----
-{
-  "executionDate": 1433980484194
-}
-----
-
-* *Success Response* +
-  The actual long value corresponding to the next execution date of the timer event trigger, as a long value
- ** *Code*: 200
- ** *Payload*:
-+
-[source,json]
-----
-{
-    "executionDate": 1433980484194
-}
-----
-+
-[#message]
-
-=== Message
-
-==== Description
-
-Use this resource to send BPM message events.
-Message events are caught by processes using `catch message event` flow nodes (Start, intermediate, boundary or receive tasks).
-
-==== Methods
-
-The methods used for this resource is:
-
-* POST - Send a message
-
-==== Send a message event
-
-* *Header* +
-`Content-Type : application/json`
-* *URL* +
-`/API/bpm/message`
-* *Method* +
-`POST`
-* *Request Payload*
-+
-[source,json]
-----
-{
-  "messageName" : "myMessage" ,
-  "targetProcess": "processName",
-  "targetFlowNode": "catchMessageFlowNodeName", //Optional
-  "messageContent" : {
-      "data1" : {
-          "value" : "aValue" //Cannot be null
-       },
-      "data2" : {
-          "value" : 42, //Cannot be null or empty
-          "type" : "java.lang.Long" //Optional
-      },
-      ...
-  },
-  "correlations" : {
-      "key1" : {
-          "value" : "aValue" //Cannot be null
-      },
-      "key2" : {
-          "value" : 123, //Cannot be null or empty
-          "type" : "java.lang.Integer" //Optional
-      },
-      ... // 5 keys max
-  }
-}
-----
-+
-_Supported value types in message content and correlations_ :
-
- ** java.lang.String
- ** java.lang.Boolean
- ** java.lang.Integer
- ** java.lang.Double
- ** java.lang.Float
- ** java.lang.Long
- ** java.util.Date (Supports *ISO-8601* format, eg: `2013-01-02T02:42:12.17+02:00`)
- ** java.time.LocalDate (Supports *ISO-8601* format, eg: `2018-02-04`)
- ** java.time.LocalDateTime (Supports *ISO-8601* format, eg: `2018-02-04T10:30:15`)
- ** java.time.OffsetDateTime (Supports *ISO-8601* format, eg: `2018-02-04T10:30:15+01:00`) +
-When not set, the type is guessed using the value. Be careful as it can lead to type inconsistency in the target process (eg: a java.lang.Long is expected and the guessed type is a java.lang.Integer)
-
-* *Success Response*
- ** *Code*: 204
+* xref:activities-tasks-flow-nodes-api.adoc[Activities, Tasks and Flow nodes]
+* xref:data-case-document-api.adoc[Data and CaseDocument]
```

Comment:
The commonly use term is simply Document and not CaseDocument, even if the technical name of the api is caseDocument:
```suggestion
* xref:data-case-document-api.adoc[Data and Document]
```